code/processes/wdb.q

```diff
@@ -17,8 +17,8 @@ upd:.wdb.replayupd;
 .wdb.clearwdbdata[];
 /- initialise the wdb process
 .wdb.startup[];
-/ - start the timer if datastriping off
-$[.ds.datastripe;.lg.o[`data;"datastriping on - savedown to ",(string .wdb.savedir)," disabled"];.wdb.saveenabled;.wdb.starttimer[]];
+/ - start the timer if datastriping off or exit process if datastriping on
+$[.ds.datastripe;[.lg.o[`data;"datastriping on - wdb will not start up "];exit 0];.wdb.saveenabled;.wdb.starttimer[]];
```

Comment:
can you just make this a `.lg.e`? that should exit proc if encountered while proc is initialising

```suggestion
$[.ds.datastripe;.lg.e[`data;"datastriping on - wdb will not start up "];.wdb.saveenabled;.wdb.starttimer[]];
```